lib/interceptor.js

```diff
@@ -263,13 +277,22 @@ var interceptor = {
       );
     }
 
+    function shouldExcludeRequestByUrl(url) {
+      return window[NAMESPACE].excludedUrls.find(function (regex) {
```

Comment:
Got it. ```find``` works because it returns undefined where the element is not in the array. ```some``` is better because I am looking to test if it is in the array AND it works with IE. I don't need the value that is in there. Thanks.